R/create_SE.R

```diff
@@ -229,9 +229,10 @@ create_SE <- function(df_,
 validate_mapping <- function(trt_df, refs_df, nested_confounders) {
   
   if (!is.null(nested_confounders)) {
-    refs_df <- refs_df[
-      refs_df[[nested_confounders]] %in% unique(trt_df[[nested_confounders]]),
-    ]
+    matching_confounders <- refs_df[[nested_confounders]] %in% unique(trt_df[[nested_confounders]])
+    if (any(refs_df[[nested_confounders]] %in% unique(trt_df[[nested_confounders]]))) {
```

Comment:
You can use `matching_confounders` variable that you declared above

Reply:
Indeed, that was a plan! Thanks!